addons/web/static/src/views/kanban/kanban_model.js

```diff
@@ -171,37 +173,35 @@ class KanbanGroup extends Group {
     }
 
     async load() {
-        const proms = [];
-        proms.push(super.load());
+        this.loadTooltip();
+        await super.load();
+    }
+
+    /**
+     * Requests the groups tooltips to the server and store them in this.tooltip
+     *
+     * @returns {Promise<void>}
+     */
+    async loadTooltip() {
         const groupName = this.groupByField.name;
         if (
             this.groupByField.type === "many2one" &&
             this.value &&
             groupName in this.model.tooltipInfo
         ) {
+            this.keepLastTooltip = this.keepLastTooltip || new KeepLast();
```

Comment:
In English, modifier nouns go before the object, eg "pencil paper" is paper to be used with pencils whereas "paper pencil" is a pencil to be used with paper. In this case, the keepLast is the object and tooltip is the modifier. A keepLastTooltip would be a tooltip related to a keepLast.